testfile

```diff
@@ -142,3 +142,5 @@ Trying to get push rules with unknown rule_id fails with 404
 Events come down the correct room
 local user can join room with version 5
 User can invite local user to room with version 5
+Should reject keys claiming to belong to a different user
+Enabling an unknown default rule fails with 404
```

Comment:
These two lines are already in the file.

Reply:
Have removed the lines.